WebRoot/public/login.jsp

```diff
@@ -652,15 +653,17 @@ if (application.getInitParameter("offlineMode") != null) {
                                             </select>
                                         </c:otherwise>
                                     </c:choose>
-                                    <script TYPE="text/javascript">
-                                        document.write("<a href='#' onclick='showWhatsThis();' id='ZLoginWhatsThisAnchor' aria-controls='ZLoginWhatsThis' aria-expanded='false'><img src='../img/questionMark.png' width='13' height='13' border='0'></a>");
-                                    </script>
+                                    <div class="alignWhatsThis">
+                                       <input type="button" onclick="document.getElementById('ZLoginWhatsThis').style.display='block'" id='ZLoginWhatsThisButton'/>
+                                    </div>
                                 </div>
                          
                            
-                                <div id="ZLoginWhatsThis" class="ZLoginInfoMessage" style="display:none;" role="tooltip">
-                                    <div id="dialogCloseButton" onclick='showWhatsThis();'><img  src="../img/close.png" width="10" height="10" ></div>
-                                    <fmt:message key="clientWhatsThisMessageWithoutTablet"/>
+                                <div id="ZLoginWhatsThis" style="display: none;">
+                                    <div class="ZLoginInfo">
+                                        <span id="dialogCloseButton" onclick="document.getElementById('ZLoginWhatsThis').style.display='none'">&times;</span>
```

Comment:
create a function for onclick handler instead of writing inline code

Reply:
Done